<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindEventDialog</class>
 <widget class="QDialog" name="BindEventDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Прив'язати прийом до картки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/action_icons/bind_event.png</normaloff>:/action_icons/bind_event.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Пошук за:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="search_cb"/>
     </item>
     <item>
      <widget class="QLineEdit" name="search_le">
       <property name="placeholderText">
        <string>Пошук...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="patients_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_pb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pb">
       <property name="text">
        <string>Скасувати</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
